index.html

```diff
@@ -1,29 +1,183 @@
+        nullam ac tortor vitae purus faucibus ornare suspendisse. Mi eget mauris pharetra et ultrices neque. 
+        Nisl vel pretium lectus quam id leo in vitae turpis. Ac felis donec et odio pellentesque diam volutpat.
+         Urna nec tincidunt praesent semper feugiat nibh sed. Adipiscing at in tellus integer. Etiam tempor 
+         orci eu lobortis elementum nibh tellus molestie nunc. Sed id semper risus in hendrerit. Mi proin sed 
+         libero enim sed faucibus. Massa tempor nec feugiat nisl pretium fusce id.
+        </p>
+        <a href="/page"><button>Learn More</button></a>
+      </div>
+    </section>
+
+
+    <!--Logos-->
+    <section class="logos">
+      <div><img class="logo1" src="photoshoplogo.png" alt="Adobe Logo"></div>
+      <div><img class="logo2" src="cpslogo.png" alt="Clip Studio Paint Logo"></div>
+      <div><img class="logo3" src="blenderlogo.png" alt="Blender Logo"></div>
+    </section>
+
+
+    <!--Education-->
+    <section class="section2">
+      <div class="section2text">
+        <h2>Education</h2>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor 
+          incididunt ut labore et dolore magna aliqua. Viverra mauris in aliquam sem fringilla ut morbi tincidunt 
+          augue. Leo vel orci porta non pulvinar. Convallis convallis tellus id interdum velit laoreet id donec. 
+          Et leo duis ut diam quam. Turpis tincidunt id aliquet risus feugiat in ante metus. Suspendisse potenti 
+          nullam ac tortor vitae purus faucibus ornare suspendisse. Mi eget mauris pharetra et ultrices neque. 
+          Nisl vel pretium lectus quam id leo in vitae turpis. Ac felis donec et odio pellentesque diam volutpat.
+           Urna nec tincidunt praesent semper feugiat nibh sed. Adipiscing at in tellus integer. Etiam tempor 
+           orci eu lobortis elementum nibh tellus molestie nunc. Sed id semper risus in hendrerit. Mi proin sed 
+           libero enim sed faucibus. Massa tempor nec feugiat nisl pretium fusce id.
+        </p>
+        <a href="/page"  id="Contact" ><button>Learn More</button></a>
+      </div>
+      <div class="section2picture">
+        <img src="university.jfif" alt="Logo">
+      </div>
+    </section>
+
+
+    <!--Contact Form-->
+    <section class="contact">
+      <div class="form-outline">
+        <h2>Contact Me</h2>
+        <div class="contactgrid">
+          <div class="contact1">
+            <input type="text" placeholder="First Name">
+            <input type="text" placeholder="Last Name">
+            <input type="text" placeholder="Email">
+            <input type="text" placeholder="Phone">
+          </div>
+          <div class="contact2">
+            <input type="text" placeholder="Address">
+            <input type="text" placeholder="Type your message here">
+            <div class="buttoncontainer">
+              <button class="button" type="submit">Submit</button>
+            </div>

```

Comment:
This form is missing a but of structure. 

You need a `form` element and then you need labels for your form. 

Would recommend looking at the different form exercises we did to just brush up on proper form markup.